changelog/1.21.0.md

```diff
@@ -48,6 +49,19 @@ There are no breaking changes in 1.21.0.
   and maximum disk size constraint to 8 TiB.
 - web: Added ability to choose the SSH keygen algorithm Coder uses when
   generating SSH keys.
+- infra: Added connection cache based on open DataChannels.
+- infra: Added option to opt-in to OIDC's new refresh feature, allowing
+  Coder-issued API keys to inherit session timing limits and to use refresh
+  tokens to ensure continued access.
+- infra: Improved speed of metrics query to improve metrics page performance.
+- infra: Added `coderd.trustProxyIP` to Helm chart to allow configuration of
+  Real IP.
+- web: Added ability to enable/disable SSH access to the edit workspace
```

Comment:
This is called "External Connect" and blocks SSH and external tunneling (except for devurls)